openapi.yaml

```diff
@@ -25417,409 +27154,1069 @@ components:
+          pattern: '[a-zA-Z0-9-_ \.]{3,64}'
+          description: >
+            The label for this Service. This is for display purposes only.
+          example: prod-1
+          x-linode-cli-display: 4
+        address:
+          type: string
+          format: url
+          minLength: 3
+          maxLength: 100
+          description: >
+            The URL at which this Service is monitored.
+
+
+            URL parameters such as `?no-cache=1` are preserved.
+
+
+            URL fragments/anchors such as `#monitor` are **not** preserved.
+          example: https://example.org
+          x-linode-cli-display: 5
+        timeout:
+          type: integer
+          minimum: 1
+          maximum: 255
+          description: >
+            How long to wait, in seconds, for a response before considering the
+            Service to be down.
+          example: 30
+        body:
+          type: string
           nullable: true
+          minLength: 0
+          maxLength: 100
+          description: >
+            What to expect to find in the response body for the Service to be
+            considered up.
+          example: it worked
+        consultation_group:
           type: string
-          minLength: 2
+          minLength: 0
           maxLength: 50
           description: >
-            A grouping for this Contact. This is for display purposes only.
-          example: on-call
-          x-linode-cli-display: 5
-        updated:
+            The group of ManagedContacts who should be notified or consulted
+            with when an Issue is detected.
+          example: on-call
+          x-linode-cli-display: 6
+        notes:
+          type: string
+          nullable: true
+          description: >
+            Any information relevant to the Service that Linode special forces
+            should know when attempting to resolve Issues.
+          example: The service name is my-cool-application
+        region:
+          type: string
+          description: >
+            The Region in which this Service is located. This is required if
+            address is a private IP, and may not be set otherwise.
+          example: null
+        credentials:
+          type: array
+          items:
+            type: integer
+            example: 9991
+          description: >
+            An array of ManagedCredential IDs that should be used when attempting to
+            resolve issues with this Service.
+        created:
           type: string
           format: date-time
-          description: >
-            When this Contact was last updated.
+          description: When this Managed Service was created.
           example: '2018-01-01T00:01:01'
           readOnly: true
-    ManagedCredential:
+        updated:
+          type: string
+          format: date-time
+          description: When this Managed Service was last updated.
+          example: '2018-03-01T00:01:01'
+          readOnly: true
+    aclb_LoadBalancer:
```

Comment:
Could these components be renamed to use full pascal case? This will make things a tiny bit easier in our code generation tooling.

e.g. `ACLBLoadBalancer`

Reply:
@lgarber-akamai, sure, I will rename the schemas for the next update.